cargo-n64/src/ipl3.rs

```diff
@@ -78,6 +80,27 @@ impl IPL3 {
         let mut ipl = [0; IPL_SIZE];
         f.read_exact(&mut ipl)?;
 
+        Self::check(ipl)
+    }
+
+    crate fn read_from_rom(path: impl AsRef<Path>) -> Result<IPL3, IPL3Error> {
+        let mut f = File::open(&path)?;
+        f.seek(SeekFrom::Start(HEADER_SIZE as u64))?;
+
+        let mut ipl = [0; IPL_SIZE];
+
+        if f.read(&mut ipl)? != IPL_SIZE {
+            return Err(IPL3Error::IPL3ReadError(format!(
+                "Couldn't read full all {} IPL3 bytes from ROM \"{}\".",
```

Comment:
nitpick on the error message, there's a grammatical error.

```suggestion
                "Couldn't read all {} IPL3 bytes from ROM \"{}\".",
```